lib/PearlBee/Comments/Builtin/AddComment.pm

```diff
@@ -0,0 +1,100 @@
+package PearlBee::Comments::Builtin::AddComment;
+use Dancer2 appname => 'PearlBee';
+use Dancer2::Plugin::DBIC;
+
+post '/comment/add' => sub {
```

Comment:
We missed this in our cleanup, for some reason... The code needs some love, but I'm not sure we care at this point... Since we're not going to use it anyway...